packaging/installer/kickstart.sh

```diff
@@ -845,7 +845,7 @@ try_package_install() {
       pkg_vsep="_"
       pkg_install_opts="${interactive_opts}"
       repo_update_opts="${interactive_opts}"
-      uninstall_subcmd="uninstall"
+      uninstall_subcmd="remove"
```

Comment:
**I didn't really check/understand** that part of the code that uses "uninstall_subcmd", asking because I noticed we do `dpkg -l <NAME>` (`apt-get remove` doesn't remove an entry from `dpkg -l`.

Reply:
The `dpkg -l` check is checking for an installed package. When invoked with a package name, it only shows the package _if_ it’s installed (and, more importantly, has an exit code of 0 if the package is installed, or 1 if it isn’t).

The whole `purge` versus `remove` thing shouldn’t matter here as we are not putting in anything registered as a config file, and even if we were there would not be time for the user to modify it before we remove it (and APT only preserves config files that have been modified).

Reply:
> , it only shows the package if it’s installed (and, more importantly, has an exit code of 0 if the package is installed, or 1 if it isn’t).

Perhaps I'm missing something, but a quick test reveals that the exit code is 0:

<details>
<summary>the test</summary>


 - check
```cmd
# dpkg -l netdata-repo && echo -e "\nEXIT CODE: $?"
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version      Architecture Description
+++-==============-============-============-=================================================================
ii  netdata-repo   1-1          all          Configuration for the official Netdata Stable package repository.

EXIT CODE: 0
```

 - remove
```cmd
# apt-get remove netdata-repo -y
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following packages will be REMOVED:
  netdata-repo
0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
After this operation, 20.5 kB disk space will be freed.
(Reading database ... 49553 files and directories currently installed.)
Removing netdata-repo (1-1) ...

```

 - check again
```cmd
# dpkg -l netdata-repo && echo -e "\nEXIT CODE: $?"
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version      Architecture Description
+++-==============-============-============-=================================================================
rc  netdata-repo   1-1          all          Configuration for the official Netdata Stable package repository.

EXIT CODE: 0
```

</details>

Reply:
Hmm, that should not be tracking those as config files...